lib/ansible/modules/cloud/digital_ocean/digital_ocean.py

```diff
@@ -346,10 +346,9 @@ def getkeyordie(k):
             module.fail_json(msg='Unable to load %s' % k)
         return v
 
-    try:
-        api_token = module.params['api_token'] or os.environ['DO_API_TOKEN'] or os.environ['DO_API_KEY']
-    except KeyError as e:
-        module.fail_json(msg='Unable to load %s' % e.message)
+    api_token = module.params['api_token'] or os.environ.get('DO_API_TOKEN') or os.environ.get('DO_API_KEY')
```

Comment:
There is another way to handle this using 
```
fallback=(env_fallback, ['DO_API_TOKEN', 'DO_API_KEY', 'DO_OAUTH_TOKEN'])
```

Check following - https://github.com/ansible/ansible/pull/29179/files#diff-9bd9b34e4d3833b922cb125ccc6085f7R193

Reply:
Hi @Akasurde I updated this. Can you please review when you get a chance. Thanks.